### PR TITLE
changed from search within title to search within lyrics

### PR DESCRIPTION
**Pull request by acquayefrank**

### CODE REVIEW

**Comment by swapagarwal** [comment]
There are a lot of files changed. Ideally, it should be a one-liner change in one file only!

File: modules/__init__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import os
 import requests
 import sys
-from src import *
+from .src import *
```
Comment (swapagarwal) [inline modules/__init__.py]: Why `.`?

File: templates/button.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from copy import deepcopy as copy
 
-from text import TextTemplate
+from .text import TextTemplate
```
Comment (swapagarwal) [inline templates/button.py]: same as above

File: templates/generic.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 from copy import deepcopy as copy
 
-from button import ButtonTemplate
+from .button import ButtonTemplate
```
Comment (swapagarwal) [inline templates/generic.py]: same as above

File: .gitignore [hunk 4]
```diff
@@ -60,3 +60,4 @@ target/
 
 #Ipython Notebook
 .ipynb_checkpoints
+*.sqlite
```
Comment (swapagarwal) [inline .gitignore]: remove this for now, to keep it clean

File: templates/generic.py (hunk 3 above)
Comment (acquayefrank) [inline templates/generic.py]: It's mostly a python-3 thing http://stackoverflow.com/questions/12172791/changes-in-import-statement-python3

File: .gitignore (hunk 4 above)
Comment (acquayefrank) [inline .gitignore]: The caching syetem is creating the .sqlite file. Or should I just ignore it locally.

File: templates/generic.py (hunk 3 above)
Comment (swapagarwal) [inline templates/generic.py]: Jarvis runs on Python 2.7

File: .gitignore (hunk 4 above)
Comment (swapagarwal) [inline .gitignore]: Ignore it locally for now, to keep the PR clean for others as reference.

**Comment by swapagarwal** [comment]
Added in #192

**Pull request closed without merging**